### ISSUE TITLE
Allow notifications of apps

### ISSUE THREAD

**Issue by AntoineTurmel** [autonome/standalone#3]
Sites like Skype or Wunderlist uses notifications, it would be great to add them in the manifest ;)

**Comment by autonome** [comment]
Not sure what you mean? Allow them to use notifications?

**Comment by AntoineTurmel** [comment]
Yes. In terms of UI it could be a popup allowing the user to set permissions of the Standalone app.

**Comment by autonome** [comment]
Hm, yeah that's going to take some doing. I wonder what Firefox permissions
do when user is fullscreen, and if that might happen here already.

I don't have time right now to do this, but you could test by requesting
notification permissions from a page, and making it an app with Standalone,
and see what happens?

**Comment by AntoineTurmel** [comment]
It's not working, but I might just contribute to Standalone ;)

**Comment by filipesizilio** [comment]
I've the same issue here with whatsapp webapp. no time to enable the notification, the popup disappear very fast.

**Comment by autonome** [comment]
If y'all want to investigate, I'm glad to review your changes and add this
feature! I just don't have time to work on it right now.

**Comment by AntoineTurmel** [comment]
@filipesizilio 
try my version if you want:
https://github.com/AntoineTurmel/standalone/releases

**Comment by filipesizilio** [comment]
@AntoineTurmel 
ok, I'm already testing. thank you.

**Comment by filipesizilio** [comment]
Hi @AntoineTurmel, until now, works fine on ubuntu 15.04 and firefox 40.0.3. Congratulations.

**Comment by autonome** [comment]
Awesome, send a PR :D

**Comment by AntoineTurmel** [comment]
@autonome 
yeah but I have to work a bit more on this, as this commit allow notifications for every website, IMHO we should ask the user before (like Firefox Desktop/Firefox OS/Android does)

**Comment by autonome** [comment]
ah yeah, good for prototype but maybe not for release.

I wonder what the doorhanger prompt does when there's no XUL window...

**Comment by nebulade** [comment]
Is the prompt really needed, since one already approves the webapp to some extent by promoting it to a desktop app. Having them enabled now and adding an improved version later is imho better than not having them at all :-) Thanks for making them work in the first place!

**Comment by autonome** [comment]
I agree that we should allow notifications by default, instead of forcing explicit user approval like a web *site*.

Apps can provide their own notification management in user preferences... like native desktop apps do!
